components/formats-bsd/src/loci/formats/FilePattern.java

```diff
@@ -723,7 +897,11 @@ private void buildFiles(String prefix, int ndx, List<String> fileList) {
     return last;
   }
 
-  /* @deprecated */
+  /**
+   * Gets the STEP element for all blocks.
+   *
+   * @return an array containing all STEP elements.
+   */
```

Comment:
Actually they were *not* deprecated, since `/* @deprecated */` is not a javadoc, so i just removed some misleading comments and added missing javadocs.